Blockzilla/Info.plist

```diff
@@ -37,13 +37,6 @@
 	<false/>
 	<key>LSApplicationCategoryType</key>
 	<string></string>
-	<key>LSApplicationQueriesSchemes</key>
-	<array>
-		<string>org-appextension-feature-password-management</string>
-		<string>googlechromes</string>
-		<string>googlechrome</string>
-		<string>firefox</string>
```

Comment:
@mintbomb27 I'm sorry this was not clear from the original issue, but `firefox` needs to say in this list. We use it to find out if Firefox is installed (through `UIApplication.canOpenURL()` so that we conditionally show the _Open in Firefox_ page action menu item. Would you mind taking out just the first three?

Reply:
Looks like we also have an _Open in Chrome_ menu item so this is really about removing the leftover `org-appextension-feature-password-management` one :-/

Reply:
Sure! No problem :)